core/src/components/modal/modal.tsx

```diff
@@ -95,7 +97,7 @@ export class Modal implements ComponentInterface, OverlayInterface {
    * array must be the value of the `initialBreakpoint` property.
    * For example: [0, .25, .5, 1]
    */
-  @Prop() breakpoints?: number[];
+  @Prop({ mutable: true }) breakpoints?: number[];
```

Comment:
Should we have an internal `sortedBreakpoints` array instead of setting `mutable: true` here? We don't necessarily need to re-render the modal whenever the breakpoints are sorted.